src/service/viewport-impl.js

```diff
@@ -191,6 +191,17 @@ export class Viewport {
   }
 
   /**
+   * Sets the body border-bottom to the specified value.
+   * @param {number} borderBottom
+   */
+  setBorderBottom(borderBottom){
```

Comment:
We should name this `#updateBorderBottom` to match the `#updatePaddingTop` we already have.